src/client/Microsoft.Identity.Client/AuthenticationResultMetadata.cs

```diff
@@ -39,5 +41,16 @@ public AuthenticationResultMetadata(TokenSource tokenSource)
         /// Time, in milliseconds, MSAL spent for HTTP communication during this request.
         /// </summary>
         public long DurationInHttpInMs { get; set; }
+
+        /// <summary>
+        /// Time, in milliseconds, remaining before the token will be proactively refreshed.
+        /// This value may be null.
```

Comment:
```suggestion
        /// Specifies the time when the cached token should be proactively refreshed.
        /// This value may be null if proactive refresh is not enabled.
```
Not a 100% on the exact wording though.

Reply:
I think this correct